tools/congestion-model/src/model/transaction.rs

```diff
@@ -11,11 +11,15 @@ pub(crate) struct Transaction {
 
     /// Where the transaction is converted to the first receipt.
     pub(crate) sender_shard: ShardId,
+    /// Where the transaction's first receipt is sent to.
+    pub(crate) receiver_shard: ShardId,
```

Comment:
Yes more consistent naming sounds reasonable :D
=> renamed as suggested

> there can be multiple receivers even within the same block right?

Not quite sure if I understand you right... But I guess the confusion was that this is only for the initial receipt?

In Near proper, the initial transaction has one sender and one receiver account id. Here we reduce it to just a shard id but otherwise it's the same fields in spirit. And there is only one receiver for a `struct Transaction` in nearcore. Though the first receipt can spawn multiple outgoing receipts with different receivers. But in the first instance, it's only one receipt and hence only one receiver. (And just to complete the picture, you can stack multiple actions inside the single receipt, they will then all go to the same receiver and are executed atomically, as in either all successful or all are aborted.)